tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java

```diff
@@ -17,9 +17,7 @@
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 import io.trino.tempto.AfterMethodWithContext;
-import io.trino.tempto.AfterTestWithContext;
```

Comment:
Should the maven compiler throws unused import error ?

Reply:
in trino checkstyle is responsible for that

Reply:
Any reason for not using checkstyle plugin here ?

Reply:
no reason that i am aware of